--- kit/Kit.cpp	2017-03-28 15:47:27.069005595 +0200
+++ kitKit.cpp	2017-03-28 15:48:40.740267751 +0200
@@ -128,6 +128,66 @@
             return true;
         }
     }
+    
+    int filecopy(FILE *src, FILE *dest)
+	{
+		const int size = 16384;
+		char buffer[size];
+		
+		while (!feof(src))
+		{
+			int n = fread(buffer, 1, size, src);
+			if (ferror(src)) {
+				return -1;
+			}
+			fwrite(buffer, 1, n, dest);
+			if (ferror(dest)) {
+				return -1;
+			}
+		}
+		fflush(dest);
+		if (ferror(dest)) {
+				return -1;
+		}
+		return 0;
+	}
+
+	int fileopen_and_copy(const char * src, const char * dest)
+	{
+		FILE * infile  = fopen(src,  "rb");
+		FILE * outfile = fopen(dest, "wb");
+		
+		if (ferror(infile)) { return -1; }
+		if (ferror(outfile)) { return -1; }
+		
+		if (filecopy(infile, outfile) == -1) {
+			return -1;
+		}
+		
+		fclose(infile);
+		fclose(outfile);
+		return 0;
+	}
+	
+	int link_or_copy_file(const char * target, const char * linkpath) {
+		int result = -1;
+		const bool hardLinkFile = true;
+		if (hardLinkFile)  {
+			result = link(target, linkpath);
+			//result = symlink(target, linkpath);
+			// continue when the file/symlink already exists
+			if ((result == -1) && (errno == EEXIST)) {
+				Log::trace("link(\"" + std::string(target) + "\",\"" + linkpath +
+			               "\") already exists. Continuing.");
+				return 0;
+			}
+		}
+		else {
+			result = fileopen_and_copy(target, linkpath);
+		}
+		return result;
+	}
+ 
 
     int linkOrCopyFunction(const char *fpath,
                            const struct stat* /*sb*/,
@@ -146,7 +206,7 @@
         case FTW_F:
         case FTW_SLN:
             File(newPath.parent()).createDirectories();
-            if (link(fpath, newPath.toString().c_str()) == -1)
+            if (link_or_copy_file(fpath, newPath.toString().c_str()) == -1)
             {
                 LOG_SYS("link(\"" << fpath << "\", \"" <<
                         newPath.toString() << "\") failed. Will copy.");
