--- loolwsd/Makefile.am	2016-08-18 18:06:04.931502710 +0200
+++ loolwsd/Makefile.am	2016-08-18 18:07:51.615114513 +0200
@@ -160,7 +160,7 @@
 # installing the RPM or Debian package.
 
 all-local: loolforkit @JAILS_PATH@ $(SYSTEM_STAMP)
-	if test "$$BUILDING_FROM_RPMBUILD" != yes; then \
+	if test 0 == 1; then \
 	    sudo @SETCAP@ cap_fowner,cap_mknod,cap_sys_chroot=ep loolforkit; \
 	    sudo @SETCAP@ cap_sys_admin=ep loolmount; \
 	    echo "Set required capabilities"; \
